python/README.md

```diff
@@ -55,81 +58,90 @@ To get description of all command-line parameters:
+$ python3 recognize_stream.py -e LINEAR16 --interim_results -r 16000 -c 1 --silence_duration_threshold 1.2 pyaudio:
 ```
 
-To get description of all command-line parameters:
+Return just the first recognized utterance and halt:
 
 ```
-$ python3 synthesize_stream.py --help
+$ python3 recognize_stream.py -e LINEAR16 --interim_results -r 16000 -c 1 --single_utterance pyaudio:
 ```
 
-### Advanced examples
+### Speech synthesis examples
 
-Use `RAW_OPUS` for both synthesis and recognition to save bandwidth:
+To run basic speech synthesis and save result to wav:
 
 ```
-$ python3 synthesize_stream.py -r 48000 -e RAW_OPUS "Газета Times, 03 января 2009 года - Канцлер на грани ради второго спасения банков." output.raw_opus
+$ python3 synthesize_stream.py -r 48000 -e LINEAR16 "И мысли тоже тяжелые и медлительные, падают неторопливо и редко одна за другой, точно песчинки в разленившихся песочных часах." output_1.wav
 ```
 
+You can also specify alternative stress with `0` sign after a vowel:
+
 ```
-$ python3 recognize_stream.py -e RAW_OPUS -r 48000 -c 1 output.raw_opus
+$ python3 synthesize_stream.py -r 48000 -e LINEAR16 "За0мок - замо0к." output_2.wav
 ```
 
-Run streaming speech recognition with interim results and disabled voice activity detection (VAD):
+Feel free to use arabic numerals and named entities:
 
 ```
-$ python3 recognize_stream.py -e MPEG_AUDIO --interim_results --do_not_perform_vad -r 16000 -c 1 ../audio/sample_1.mp3
+$ python3 synthesize_stream.py -r 48000 -e LINEAR16 "Газета Times, 03 января 2009 года - Канцлер на грани ради второго спасения банков." output_3.wav
 ```
 
-Use audio from microphone (requires `PyAudio`):
+For now, `LINEAR16` does not support samples rates other than 48kHz.
+Use `RAW_OPUS` ([opuslib](https://github.com/orion-labs/opuslib) required) to specify different sample rates:
 
 ```
-$ python3 recognize_stream.py -e LINEAR16 --interim_results -r 16000 -c 1 --silence_duration_threshold 0.3 pyaudio:
+$ python3 synthesize_stream.py -r 16000 -e RAW_OPUS "Привет, мир." output_4.wav
 ```
 
-Specify longer silence timeout for voice activity detection:
+It is possible to use [SSML](https://en.wikipedia.org/wiki/Speech_Synthesis_Markup_Language):
 
 ```
-$ python3 recognize_stream.py -e LINEAR16 --interim_results -r 16000 -c 1 --silence_duration_threshold 1.2 pyaudio:
+$ python3 synthesize_stream.py -r 48000 -e LINEAR16 --ssml "<speak><p><s>Оригинальная мысль?</s><s>Нет ничего легче.</s></p><break time='300ms'/><p><s>Библиотеки просто набиты ими.</s></p></speak>" output_5.wav
 ```
 
-Return just the first recognized utterance and halt:
+You can also specify voice name:
 
 ```
-$ python3 recognize_stream.py -e LINEAR16 --interim_results -r 16000 -c 1 --single_utterance pyaudio:
+$ python3 synthesize_stream.py -r 48000 -e LINEAR16 --voice alyona:funny "Привет! Я Алёна. Я помогу в озвучке книг, новостей, образовательных курсов, а также могу быть твоим напарником для медитации." output_6.wav
```

Comment:
It would be helpful for user to see list of all available voices somewhere.
I think python examples are a good choice for that.